core/src/main/resources/lib/layout/task.jelly

```diff
@@ -84,8 +88,33 @@ THE SOFTWARE.
     <j:set var="taskMatching" value="${context.parent}"/>
     <d:invokeBody />
   </j:scope>
+  <j:if test="${attrs.permission != null}">
+    <j:choose>
+      <j:when test="${h.hasPermission(it, attrs.permission)}">
+        <j:set var="hasPermission" value="true" />
+      </j:when>
+      <j:otherwise>
+        <j:set var="hasPermission" value="false" />
+      </j:otherwise>
+    </j:choose>
+  </j:if>
+
+  <j:if test="${attrs.permissions != null}">
+    <j:choose>
+      <j:when test="${h.hasAnyPermission(it, attrs.permissions)}">
+        <j:set var="hasPermission" value="true" />
+      </j:when>
+      <j:otherwise>
+        <j:set var="hasPermission" value="false" />
```

Comment:
Questionable semantics when both attributes are specified? Unsure. Should be documented what happens when you do both, and it should be consistent with `/lib/layout.jelly`.

Plus I wonder as before whether this could be simplified to
```suggestion
        <j:set var="hasPermission" value="${h.hasAnyPermission(it, attrs.permissions)}" />
```
plus delete a bunch of surrounding lines (obviously only if we're OK with the current behavior).